pkg/services/ngalert/api/tooling/definitions/prom.go

```diff
@@ -115,10 +115,10 @@ type Rule struct {
 	Name string `json:"name"`
 	// required: true
 	Query  string         `json:"query"`
-	Labels overrideLabels `json:"labels"`
```

Comment:
drive-by fix due to https://github.com/prometheus/prometheus/blob/main/web/api/v1/api.go#L1183-L1185